frame/vesting/src/tests.rs

```diff
@@ -545,12 +552,17 @@ fn claim_works() {
 			VestingScheduleIdSet::All
 		));
 
+		let mut amount_claimed_per_schedule: BoundedBTreeMap<_, _, MaxVestingSchedule> =
+			BoundedBTreeMap::new();
+		amount_claimed_per_schedule.try_insert(4_u128, 10).unwrap();
```

Comment:
use `[(4_u128, 10)].into_iter().try_collect()` here and other spots where you create a new map and immediately `.try_insert(..)` into it (you'll need to import `TryCollect`)